src/open_company_web/api.cljs

```diff
@@ -41,39 +41,39 @@
       (let [body (if (:success response) (json->cljs (:body response)) {})]
         (flux/dispatch dispatcher/companies body)))))
 
-(defn get-company [ticker]
+(defn get-company [symbol]
```

Comment:
right.. i'll change it back

On Fri, Aug 7, 2015 at 2:13 PM, Sean Johnson <notifications@github.com>
wrote:

> In src/open_company_web/api.cljs
> <https://github.com/open-company/open-company-web/pull/16#discussion_r36512680>
> :
>
> > @@ -41,39 +41,39 @@
> >        (let [body (if (:success response) (json->cljs (:body response)) {})]
> >          (flux/dispatch dispatcher/companies body)))))
> >
> > -(defn get-company [ticker]
> > +(defn get-company [symbol]
>
> ticker is probably better.
>
> "symbol" is a clojure core function:
> https://clojuredocs.org/clojure.core/symbol
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/open-company/open-company-web/pull/16/files#r36512680>
> .
>



-- 
ic